axon-framework/events/event-processors/streaming.md

```diff
@@ -384,12 +384,14 @@ Token stealing can, for example, happen if event processing is slow or encounter
  
 ### Initial Tracking Token
 
+> Please note that we will be referring to the oldest (very first) token as the "tail" of the stream, and the most recent (the latest) token as the "head" of the stream in this document. 
```

Comment:
I think it would be good if we add a header to this notice. Something like "Heads or Tails?".
And, I think this belongs right after the first time we refer to head/tail in this document, which I believe is completely up top.

Reply:
Agreed on both counts. Will make the adjustments. Thanks!

Reply:
With the changes made, I think we can remove this notice actually!

Reply:
Sounds good to me. I just left it there because I couldn't find a good spot to explain the "head" of the stream. But I'll go ahead and remove it. :)